### PR TITLE
Improve Store docs

### PR DESCRIPTION
**Pull request by Manifaust**
- Better introduction to video, alt text
- Better organization of deployment configuration

### CODE REVIEW

File: scst-store/deployment_details.md [hunk 1]
```diff
@@ -15,6 +15,45 @@ Each component includes:
...
+Where `PASSWORD-0123` is the same password used between deployments.
+
+> Note: there is a known issue related to changing database passwords [Known Issues - Persistent Volume Retains Data](scst-store/known_issues.md#persistent-volume-retains-data).
+
+### App Service Type
+
+If your environment does not support `LoadBalancer`, and you want to use `NodePort`, configure the `app_service_type` property in your `scst-store-values.yaml`:
+
+```yaml
+app_service_type: "LoadBalancer"
```
Comment (xtreme-jason-smith) [inline scst-store/deployment_details.md:51]: You may want to change `"LoadBalancer"` to `"NodePort"` so it's more consistent with your explination.

File: install-components.md [hunk 2]
```diff
@@ -1573,23 +1573,15 @@ To install Supply Chain Security Tools - Store:
...
-    If your environment does not support `LoadBalancer`, omit the `app_service_type` line so that
-    the default value `NodePort` is used instead.
-
-   > **Note:**
-    By default, a service account with read-write privileges to the metadata store app is installed. This service account is a cluster-wide user that uses ClusterRole.
 
-     To not have `read-write` service account installed by default,
-     set `add_default_rw_service_account` property to `"false"`. To create custom service account, see [create service account doc](scst-store/create_service_account_access_token.md).
-   - The store supports connecting to PostgreSQL Amazon RDS. Please refer to the [AWS RDS Postgres for Metadata Store docs](scst-store/use_aws_rds.md)
+    By default, a service account with read-write privileges to the Supply Chain Security Tools - Store is installed. This service account is a cluster-wide user that uses ClusterRole. A read-only service account can be created, see [Deployment Details and Configuration](scst-store/deployment_details.md#configuration).
```
Comment (xtreme-debbie-chen) [inline install-components.md:1584]: I also wrote changes around this in my PR 😕 
https://github.com/pivotal/docs-tap/pull/529

Comment (Manifaust) [reply install-components.md:1584]: I'll remove my changes

File: scst-store/deployment_details.md (hunk 1 above)
Comment (Manifaust) [reply scst-store/deployment_details.md:51]: I'll change this

File: scst-store/deployment_details.md [hunk 3]
```diff
@@ -15,6 +15,45 @@ Each component includes:
 * A Kubernetes secret to allow pulling Supply Chain Security Tools - Store images from a registry.
 * A namespace called `metadata-store`.
 
-## Database Details
+## <a id='configuration'></a> Deployment Configuration
+### Database Configuration
 
-The default database that ships with the deployment is meant to get users started with using the metadata store. The default database deployment is not meant support for many enterprise production requirements including scaling, redundancy, or failover. However, it is still a secure deployment.
\ No newline at end of file
+The default database that ships with the deployment is meant to get users started with using the metadata store. The default database deployment is not meant support for many enterprise production requirements including scaling, redundancy, or failover. However, it is still a secure deployment.
```
[the remark this answers is not in the document]
Comment (Manifaust) [reply scst-store/deployment_details.md:21]: Will update this in some future date

**Comment by RichardJJG** [comment]
Thanks, guys!

**Pull request merged**